canonical/vanilla-framework
# Add left & right chevron icon variants

## Done

Adds left & right versions of the chevron icon. These are available as `p-icon--chevron-left` and `p-icon--chevron-right`, respectively.

Fixes #5066, [WD-11767](https://warthogs.atlassian.net/browse/WD-11767)

## QA

- Open [icons page](https://vanilla-framework-5093.demos.haus/docs/patterns/icons) and verify that left and right chevrons are displayed properly.
- Open standalone icons examples that include the chevron and verify the left and right chevrons are displayed properly.
    - https://vanilla-framework-5093.demos.haus/docs/examples/patterns/icons/icons-dark-deprecated
    - https://vanilla-framework-5093.demos.haus/docs/examples/patterns/icons/icons-dark
    - https://vanilla-framework-5093.demos.haus/docs/examples/patterns/icons/icons-light
- Review [release notes](https://vanilla-framework-5093.demos.haus/docs/whats-new) for v4.12.0

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots
![image](https://github.com/canonical/vanilla-framework/assets/46915153/bcfb15a9-b246-4928-82d0-5769204104db)
![image](https://github.com/canonical/vanilla-framework/assets/46915153/68523d88-cf10-47c8-ae54-1f651f95a207)
![image](https://github.com/canonical/vanilla-framework/assets/46915153/6f009108-5d2b-41de-b9c6-ae9e00d89042)
![image](https://github.com/canonical/vanilla-framework/assets/46915153/2b5cb033-44f0-4138-bed9-1fa9720636bd)



[WD-11767]: https://warthogs.atlassian.net/browse/WD-11767?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
